styles/stylesheet.css

```diff
@@ -0,0 +1,69 @@
+*{
+border: 0;
+padding: 0;
+margin: 0;
+}
+
+
+header {
+  background-color: #1c384f;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+}
+
+.logoimg {
+  width: 40%;
+  padding: 0;
+}
+
+.headertext {
+  color: #668c2c;
+  font-style: italic;
+  font-size: 1.8rem;
+  padding-bottom: 3rem;
+}
+
+/* figure out how to float to the top right of the header */
+
+.moveit {
```

Comment:
I'd try a different way of laying out the header. Floats are a pain to get right. Try flexbox, grid or position